src/components/CurrentRefinements/CurrentRefinements.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'preact-compat';
+import PropTypes from 'prop-types';
+import isEqual from 'lodash/isEqual';
+import { isSpecialClick, capitalize } from '../../lib/utils.js';
+
+const createItemKey = ({ attribute, value, type, operator }, index) =>
+  `${[attribute, type, value, operator]
+    .map(key => key)
+    .filter(Boolean)
+    .join(':')}#${index + 1}`;
+
+const handleClick = callback => event => {
+  if (isSpecialClick(event)) {
+    return;
+  }
+
+  event.preventDefault();
+  callback();
+};
+
+class CurrentRefinements extends Component {
+  shouldComponentUpdate(nextProps) {
+    return !isEqual(this.props.refinements, nextProps.refinements);
+  }
+
+  renderRefinement(refinement, index) {
+    return (
+      <li
+        key={`${refinement.attribute}-${index}`}
+        className={this.props.cssClasses.item}
+      >
+        <span className={this.props.cssClasses.label}>
+          {capitalize(refinement.attribute)}:
+        </span>
+
+        {refinement.items.map((item, itemIndex) => (
+          <span
+            key={createItemKey(item, itemIndex)}
+            className={this.props.cssClasses.category}
+          >
+            <span className={this.props.cssClasses.categoryLabel}>
+              {item.attribute === 'query' ? <q>{item.label}</q> : item.label}
```

Comment:
We don't provide template for this widget?

Reply:
We discussed it, and the template doesn't provide anything transformItems can't

It becomes especially tricky with the nested form; where does the template go? in the inner bit means you can't customise the main label 🤔

Reply:
Right, the template doesn't really make sense here.